packages/mobile/locales/en-US/global.json

```diff
@@ -160,5 +160,6 @@
   "addressValidationPartialOwnAddress": "Please enter the last 4 digits of your contact's account number, not your own. Press the link below for help",
   "helpFindAccount": "Help your contact find their account number",
   "close": "Close",
+  "dismiss": "Dismiss",
```

Comment:
Nit, I guess we already have this here:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/locales/en-US/walletFlow5.json#L72

But global.json is a better place for it. That old one is a dumping ground before we broke these out by major flow. If you don't mind, please remove the old one.